roles/prometheus/templates/targets/zookeeper_exporter_target.json.j2

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "targets": [
+      {%- for item in groups.zookeeper -%}
+      {% if not loop.last %}
```

Comment:
why this is necessary?

Reply:
To prevent extra comma after the last element. [host1,host2,host3]